packages/trip-form/src/MetroModeSelector/SubSettingsPane.tsx

```diff
@@ -81,7 +87,8 @@ const ModeSettingRenderer = ({
   const labelWithIcon =
     "icon" in setting ? (
       <FormLabelIconWrapper>
-        <span role="none">{setting.icon}</span> {label}
+        <div role="none">{setting.icon}</div>
+        <div> {label}</div>
```

Comment:
I generally prefer `1ch` of left padding so it's not selectable

Reply:
I'll use flexbox gap